tests/ztimer_xsec/main.c

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 2021 TUBA Freiberg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup tests
+ * @{
+ *
+ * @file
+ * @brief    high level ztimer test application
+ *
+ * @author   Karl Fessel <karl.fessel@ovgu.de>
+ *
+ *
+ * @}
+ */
+
+#include <stdio.h>
+
+#include "ztimer.h"
+#include "mutex.h"
+
+#ifndef LOG_LEVEL
+#define LOG_LEVEL LOG_WARNING
+#endif
+
+#include "log.h"
+
+typedef struct named_lock {
+    char *name;
+    mutex_t mut;
+} named_lock_t;
+
+void release(void *arg);
+
+static named_lock_t sec_lock = { .name = "SEC", .mut = MUTEX_INIT_LOCKED };
+static named_lock_t msec_lock = { .name = "MSEC", .mut = MUTEX_INIT_LOCKED };
+static named_lock_t usec_lock = { .name = "USEC", .mut = MUTEX_INIT_LOCKED };
+
+static ztimer_t sec_tim = { .callback = release, .arg =  &sec_lock };
+static ztimer_t msec_tim = { .callback = release, .arg = &msec_lock };
+static ztimer_t usec_tim = { .callback = release, .arg = &usec_lock };
+void release(void *arg)
+{
+    named_lock_t *e = arg;
+
+    LOG_INFO("time: %" PRIu32 "\n", ztimer_now(ZTIMER_USEC));
+
+    printf("unlocking %s\n", e->name);
+    mutex_unlock(&e->mut);
+}
+
+int main(void)
+{
+    puts("starting ztimers");
+    /* start a timer on each high level ztimer*/
+    ztimer_set(ZTIMER_SEC,   &sec_tim,  1);
```

Comment:
```suggestion
    ztimer_set(ZTIMER_SEC, &sec_tim,  1);
```

Reply:
the spaces is there for alignment (readability) if you still want this to be gone give me a hint

Reply:
Actually it would be bets if you simply uncrustify the file `uncrustify -c uncrustify-riot.cfg tests/ztimer_xsec/main.c`, that will take care of all these comments.

Reply:
k, I it is the me vs uncrustify conflict again and for some reason it wins even though it worsens the readability

btw i had the main.c run through uncrustify but my lokal one (same config) did not criticize all the align spaces that yours found

Reply:
Maybe we should give `clang-format` a try. `uncrustify` has a lot of issues with parsing C code (e.g. confusing unary `*` operator with multiplication). I'm pretty sure that `clang-format` is not having these issues.